packages/expo-camera/ios/Current/BarcodeScanner.swift

```diff
@@ -76,6 +76,10 @@ actor BarcodeScanner: NSObject, BarcodeScanningResponseHandler {
     }
   }
 
+  func setOnBarcodeScanned(_ onBarcodeScanned: @escaping ([String: Any]?) -> Void) {
+    self.onBarcodeScanned = onBarcodeScanned
+  }
+
```

Comment:
Let's make the `onBarcodeScanned` property private now that it's not accessed directly outside of the class